packages/nextjs/cypress/e2e/contract_interaction.cy.ts

```diff
@@ -0,0 +1,719 @@
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "inputs": [],
+    "name": "validatorFee",
+    "outputs": [
+      {
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "name": "validatorList",
+    "outputs": [
+      {
+        "internalType": "address",
+        "name": "",
+        "type": "address"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [],
+    "name": "validatorListCount",
+    "outputs": [
+      {
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "_validatorAddress",
+        "type": "address"
+      }
+    ],
+    "name": "validatorRemove",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "inputs": [],
+    "name": "validatorTotalFee",
+    "outputs": [
+      {
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "uint256",
+        "name": "_fee",
+        "type": "uint256"
+      }
+    ],
+    "name": "validatorsetFee",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  }
+]`;
+
+describe("Contract Interaction", () => {
+  before(() => {
+    cy.wakeUpHeimdall();
+  });
+
+  it("should load DAI contract and interact with its balanceOf method", () => {
+    cy.visit("http://localhost:3000");
+    cy.loadContract("0x6B175474E89094C44Da98b954EedeAC495271d0F", 1000);
+    cy.url().should("include", "/0x6B175474E89094C44Da98b954EedeAC495271d0F/1");
+    cy.interactWithMethod("balanceOf", "0x6B175474E89094C44Da98b954EedeAC495271d0F");
+  });
+
+  it("should load proxy contract on Base and interact with its balanceOf method", () => {
+    cy.visit("http://localhost:3000");
+    cy.selectNetwork("Base");
+    cy.loadContract("0xca808b3eada02d53073e129b25f74b31d8647ae0", 1000);
+    cy.url().should("include", "/0xca808b3eada02d53073e129b25f74b31d8647ae0/8453");
+    cy.contains("Implementation Address").should("be.visible");
+    cy.wait(1000); // wait for: the method card to re-render
+    cy.interactWithMethod("balanceOf", "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045");
+  });
+
+  it("should load unverified contract on Sepolia and ADD changeOwner write method to the UI", () => {
+    cy.visit("http://localhost:3000");
+    cy.selectNetwork("Sepolia");
+    cy.loadContract("0x759c0e9d7858566df8ab751026bedce462ff42df", 6000);
```

Comment:
If we plan to go with https://github.com/BuidlGuidl/abi.ninja/pull/140/files#r1697899465 

We could replace this line with : 

```ts
    cy.get('input[placeholder="Contract address"]').type("0x759c0e9d7858566df8ab751026bedce462ff42df");

```

Reply:
Done! Thanks